src/logging.rs

```diff
@@ -82,16 +83,18 @@ fn add_field_and_payload(data: &mut String, field: &str, payload: &str) {
         data.push_str(&field_payload);
     }
 }
-
-/// Print a message to the journal with the given priority.
-///
+/// Send a message to the local systemd journal. 
```

Comment:
`Send a message with structured properties to the journal.`

Also, please leave a line with just `///` between the title and the rest of the docstring, rustdoc will split the body.